### PR TITLE
CCTRI-2494 - Sumo Logic unit tests

### PR DESCRIPTION
**Pull request by oshynk**
- health, auth, watchdog tests
- enrich tests
- added handling of 'NOT STARTED' Sumo api search job status

### CODE REVIEW

File: code/tests/unit/api/test_authorization.py [hunk 1]
```diff
@@ -50,50 +53,45 @@ def _make_payload_message(message):
...
-):
+        authorization_errors_expected_payload):
+
     response = client.post(route, headers=get_headers(wrong_jwt_structure))
 
     assert response.status_code == HTTPStatus.OK
     assert response.json == authorization_errors_expected_payload(
-        'Wrong JWT structure'
-    )
+        'Wrong JWT structure')
```
Comment (MaxMhonc) [inline code/tests/unit/api/test_authorization.py:85]: Maybe should import error messages from utils.py?

File: code/api/utils.py [hunk 2]
```diff
@@ -32,7 +32,6 @@ def get_public_key(jwks_host, token):
     expected_errors = {
         ConnectionError: WRONG_JWKS_HOST,
         InvalidURL: WRONG_JWKS_HOST,
-        KeyError: WRONG_JWKS_HOST,
         JSONDecodeError: WRONG_JWKS_HOST
     }
```
Comment (MaxMhonc) [inline code/api/utils.py:36]: Maybe we don't need use dict, if all values the same?

File: code/tests/unit/api/test_authorization.py (hunk 1 above)
Comment (oshynk) [reply code/tests/unit/api/test_authorization.py]: That's right. Thank you, done

File: code/api/utils.py (hunk 2 above)
Comment (oshynk) [reply code/api/utils.py]: Yes, this is updated now

**Review by MaxMhonc — APPROVED** [review]

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Pull request merged**